lib/manager/jenkins/extract.ts

```diff
@@ -0,0 +1,28 @@
+import * as datasourceJenkins from '../../datasource/jenkins';
+import { logger } from '../../logger';
+import * as dockerVersioning from '../../versioning/docker';
+import { PackageDependency, PackageFile } from '../common';
+
+export function extractPackageFile(content: string): PackageFile | null {
+  logger.trace('jenkins.extractPackageFile()');
+  const deps: PackageDependency[] = [];
+  const regex = /^\s*(?<depName>[\d\w-]+):(?<currentValue>[^#\s]+).*$/;
+
+  for (const line of content.split('\n')) {
+    const match = regex.exec(line);
+
+    if (match) {
+      const { depName, currentValue } = match.groups;
+      const dep: PackageDependency = {
+        datasource: datasourceJenkins.id,
+        versioning: dockerVersioning.id,
+        depName,
+        currentValue,
+      };
+
+      deps.push(dep);
+    }
+  }
+
+  return { deps };
```

Comment:
@rarkins Return null if deps.length == 0 ?

Reply:
It has been like that in the first version (empty check and return `null`), but @rarkins suggested having it like it is now.